controllers/eventbus/installer/nats.go

```diff
@@ -483,14 +520,18 @@ streaming {
 	node_id: $POD_NAME
 	peers: [%s]
 	log_path: /data/stan/logs
+	raft_heartbeat_timeout: "%s"
+    raft_election_timeout: "%s"
```

Comment:
minor - indent

Reply:
weird.. it looked perfectly indented on GoLand
Anyway, I've fixed it, thanks